<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWidget</class>
 <widget class="QWidget" name="ManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grpBank">
     <property name="title">
      <string>Bank</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QToolButton" name="btnAddBank">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbBanks"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnBankSettings">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnOpenExplorer">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnRemoveBank">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lblBankInfo">
          <property name="text">
           <string>Bank Information</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblSearchIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::PlainText</enum>
       </property>
       <property name="pixmap">
        <pixmap resource="../../data/HyEditor.qrc">:/icons16x16/search.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSearchText">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSearch"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkShowAllBanks">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Show All Banks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ManagerTreeView" name="assetTree">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QToolButton" name="btnGenerateAsset">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCreateFilter">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnImportAssets">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnImportDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnReplaceAssets">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDeleteAssets">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDeleteAssets">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Removes the selected item(s). All affected banks will be regenerated.</string>
   </property>
  </action>
  <action name="actionReplaceAssets">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/atlas-replaceFrame.png</normaloff>:/icons16x16/atlas-replaceFrame.png</iconset>
   </property>
   <property name="text">
    <string>Replace Assets</string>
   </property>
   <property name="toolTip">
    <string>Replace the selected assets(s). All affected banks will be regnerated.</string>
   </property>
  </action>
  <action name="actionAddFilter">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Filter-New.png</normaloff>:/icons16x16/items/Filter-New.png</iconset>
   </property>
   <property name="text">
    <string>Add Filter</string>
   </property>
   <property name="toolTip">
    <string>Insert a new filter to organize assets</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-rename.png</normaloff>:/icons16x16/generic-rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Renames the selected filter or asset</string>
   </property>
  </action>
  <action name="actionAddBank">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>Add Bank</string>
   </property>
   <property name="toolTip">
    <string>Creates a new asset bank</string>
   </property>
  </action>
  <action name="actionRemoveBank">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Bank</string>
   </property>
   <property name="toolTip">
    <string>Removes the currently selected bank.</string>
   </property>
  </action>
  <action name="actionImportAssets">
   <property name="icon">
    <iconset>
     <normaloff>:/icons16x16/items/Atlas-New.png</normaloff>:/icons16x16/items/Atlas-New.png</iconset>
   </property>
   <property name="text">
    <string>Import Assets</string>
   </property>
   <property name="toolTip">
    <string>Import 1 or more assets from a single directory</string>
   </property>
  </action>
  <action name="actionImportDirectory">
   <property name="icon">
    <iconset>
     <normaloff>:/icons16x16/items/Atlas-Open.png</normaloff>:/icons16x16/items/Atlas-Open.png</iconset>
   </property>
   <property name="text">
    <string>Import Asset Directory</string>
   </property>
   <property name="toolTip">
    <string>Imports all assets found recursively from a start directory</string>
   </property>
  </action>
  <action name="actionBankSettings">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-settings.png</normaloff>:/icons16x16/generic-settings.png</iconset>
   </property>
   <property name="text">
    <string>Bank Settings</string>
   </property>
   <property name="toolTip">
    <string>Change asset bank options and for the currently selected bank</string>
   </property>
  </action>
  <action name="actionAssetSettings">
   <property name="icon">
    <iconset>
     <normaloff>:/icons16x16/items/Atlas-Settings.png</normaloff>:/icons16x16/items/Atlas-Settings.png</iconset>
   </property>
   <property name="text">
    <string>Asset Properties</string>
   </property>
   <property name="toolTip">
    <string>Configure all selected assets' properties</string>
   </property>
  </action>
  <action name="actionGenerateAsset">
   <property name="icon">
    <iconset>
     <normaloff>:/icons16x16/items/Atlas-New.png</normaloff>:/icons16x16/items/Atlas-New.png</iconset>
   </property>
   <property name="text">
    <string>Generate Asset</string>
   </property>
   <property name="toolTip">
    <string>Add a new file to project</string>
   </property>
  </action>
  <action name="actionCreateTileSet">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/TileSet-New.png</normaloff>:/icons16x16/items/TileSet-New.png</iconset>
   </property>
   <property name="text">
    <string>Create New TileSet</string>
   </property>
   <property name="toolTip">
    <string>TileSets are a group of image tiles to create a game's layout and are used by TileMaps</string>
   </property>
  </action>
  <action name="actionOpenBankExplorer">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Prefix.png</normaloff>:/icons16x16/items/Prefix.png</iconset>
   </property>
   <property name="text">
    <string>Open Bank in Explorer</string>
   </property>
   <property name="toolTip">
    <string>Opens File Explorer to the currently selected bank</string>
   </property>
  </action>
  <action name="actionOpenAssetExplorer">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Prefix-Open.png</normaloff>:/icons16x16/items/Prefix-Open.png</iconset>
   </property>
   <property name="text">
    <string>Open Containing Meta Folder</string>
   </property>
   <property name="toolTip">
    <string>Opens File Explorer to the copy of the asset's meta file</string>
   </property>
  </action>
  <action name="actionSyncFilterAssets">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/sync-folder.png</normaloff>:/icons16x16/sync-folder.png</iconset>
   </property>
   <property name="text">
    <string>Synchronize Filter Assets</string>
   </property>
   <property name="toolTip">
    <string>Synchronize a filters' assets with how they are stored on the file system</string>
   </property>
  </action>
  <action name="actionNewBuild">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Build-New.png</normaloff>:/icons16x16/items/Build-New.png</iconset>
   </property>
   <property name="text">
    <string>New Build</string>
   </property>
   <property name="toolTip">
    <string>Create a new build for the currently active project</string>
   </property>
  </action>
  <action name="actionBuildSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Build-Settings.png</normaloff>:/icons16x16/items/Build-Settings.png</iconset>
   </property>
   <property name="text">
    <string>Build Settings</string>
   </property>
   <property name="toolTip">
    <string>Change build settings for currently active project</string>
   </property>
  </action>
  <action name="actionOpenBuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Build-Open.png</normaloff>:/icons16x16/items/Build-Open.png</iconset>
   </property>
   <property name="text">
    <string>Open Build</string>
   </property>
   <property name="toolTip">
    <string>Launches the IDE or equivalent of this build</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDeleteBuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Build-Delete.png</normaloff>:/icons16x16/items/Build-Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Build</string>
   </property>
   <property name="toolTip">
    <string>Deletes the selected build (this does not affect source code)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionPackageBuild">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/package-x-generic.png</normaloff>:/icons16x16/package-x-generic.png</iconset>
   </property>
   <property name="text">
    <string>Package Build</string>
   </property>
   <property name="toolTip">
    <string>Create a redistributable package of this game, using this build</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ManagerTreeView</class>
   <extends>QTreeView</extends>
   <header>ManagerWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../data/HyEditor.qrc"/>
  <include location="../../data/HyEditor.qrc"/>
 </resources>
 <connections/>
</ui>
